ftp/1.5/modules/ROOT/pages/ftp-connection.adoc

```diff
@@ -0,0 +1,36 @@
+= Connect to an FTP Server Example - Mule 4
+
+To configure the global element to connect to an FTP Server, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to *Connector configuration* to access the global element configuration fields.
+. On the *General* tab, specify the connection information for the connector.
+. Set *Working Directory* to `${workingDir}`. +
```

Comment:
If you put the plus sign alone on a line, antora will create a blank line. which is what you want here and in other equivalent places.

Reply:
Yes:
```suggestion
. Set *Working Directory* to `${workingDir}`. +
+
```